src/Riok.Mapperly/Descriptors/MappingBuilders/InlineExpressionMappingBuilder.cs

```diff
@@ -25,21 +25,51 @@ public static class InlineExpressionMappingBuilder
         }
 
         var methodSyntax = methodSyntaxRef.GetSyntax();
-        if (methodSyntax is not MethodDeclarationSyntax { ExpressionBody: { } body, ParameterList.Parameters: [var sourceParameter] })
+
+        var bodyExpression = methodSyntax switch
+        {
+            // => expression
+            MethodDeclarationSyntax { ExpressionBody: { } body, ParameterList.Parameters: [var sourceParameter1] } => body.Expression,
```

Comment:
`ParameterList.Parameters: [var sourceParameter1]` could be simplified to `ParameterList.Parameters.Count: 1`

Reply:
Oops, this is not need at all (copy/paste left-over).